src/loader.ts

```diff
@@ -218,7 +218,7 @@ async function resolveConfig (source: string, options: LoadConfigOptions): Promi
   if (isDir) { source = options.configFile; }
   const res: ResolvedConfig = { config: undefined, cwd };
   try {
-    res.configFile = options.jiti.resolve(resolve(cwd, source), { paths: [cwd] });
+    res.configFile = resolve(options.jiti.resolve(resolve(cwd, source), { paths: [cwd] }));
```

Comment:
Can you please check result of `jiti.resolve()`? It is strange if it doesn't normalize windows path as is using pathe itself (https://github.com/unjs/jiti/blob/main/src/jiti.ts)

Reply:
For some reason it doesn't normalize path in all cases

```js
{                                                                                                     
  cwd: 'C:/Users/kozac/AppData/Local/Temp/c12/github_unjs_c12/test/fixture',                          
  source: 'config',                                                                                   
  'resolve(cwd, source)': 'C:/Users/kozac/AppData/Local/Temp/c12/github_unjs_c12/test/fixture/config',
  'options.jiti.resolve': 'C:\\Users\\kozac\\AppData\\Local\\Temp\\c12\\github_unjs_c12\\test\\fixture\\config.ts',
  'resolve(options.jiti.resolve)': 'C:/Users/kozac/AppData/Local/Temp/c12/github_unjs_c12/test/fixture/config.ts'
}
{
  cwd: 'C:/Users/kozac/AppData/Local/Temp/c12/github_unjs_c12/test/fixture',
  source: './config.dev',
  'resolve(cwd, source)': 'C:/Users/kozac/AppData/Local/Temp/c12/github_unjs_c12/test/fixture/config.dev',
  'options.jiti.resolve': 'C:\\Users\\kozac\\AppData\\Local\\Temp\\c12\\github_unjs_c12\\test\\fixture\\config.dev.ts',
  'resolve(options.jiti.resolve)': 'C:/Users/kozac/AppData/Local/Temp/c12/github_unjs_c12/test/fixture/config.dev.ts'
}
{
  cwd: 'C:/Users/kozac/AppData/Local/Temp/c12/github_unjs_c12/test/fixture/node_modules/c12-npm-test',
  source: 'C:\\Users\\kozac\\AppData\\Local\\Temp\\c12\\github_unjs_c12\\test\\fixture\\node_modules\\c12-npm-test\\config.ts',
  'resolve(cwd, source)': 'C:/Users/kozac/AppData/Local/Temp/c12/github_unjs_c12/test/fixture/node_modules/c12-npm-test/config.ts',
  'options.jiti.resolve': 'C:/Users/kozac/AppData/Local/Temp/c12/github_unjs_c12/test/fixture/node_modules/c12-npm-test/config.ts',
  'resolve(options.jiti.resolve)': 'C:/Users/kozac/AppData/Local/Temp/c12/github_unjs_c12/test/fixture/node_modules/c12-npm-test/config.ts'
}
```

Reply:
Difference in thehe cases in extensions. 1-2 can't be resolved by `resolvePathSync` ( TypeError [ERR_INVALID_URL_SCHEME]: The URL must be of scheme file) and resolving by `nativeRequire.resolve`. 
But last one resolving by `resolvePathSync` from `mlly`.

```js
const jiti = require('.')(undefined, {
    esmResolve: true
})

console.log({
    'C:/Users/kozac/AppData/Local/Temp/c12/github_unjs_c12/test/fixture/config.ts': JSON.stringify(jiti.resolve('C:/Users/kozac/AppData/Local/Temp/c12/github_unjs_c12/test/fixture/config.ts', { paths: [cwd] })),
    'C:/Users/kozac/AppData/Local/Temp/c12/github_unjs_c12/test/fixture/config': JSON.stringify(jiti.resolve('C:/Users/kozac/AppData/Local/Temp/c12/github_unjs_c12/test/fixture/config', { paths: [cwd] })),
})
```

```js
{
  'C:/Users/kozac/AppData/Local/Temp/c12/github_unjs_c12/test/fixture/config.ts': '"C:/Users/kozac/AppData/Local/Temp/c12/github_unjs_c12/test/fixture/config.ts"',
  'C:/Users/kozac/AppData/Local/Temp/c12/github_unjs_c12/test/fixture/config': '"C:\\\\Users\\\\kozac\\\\AppData\\\\Local\\\\Temp\\\\c12\\\\github_unjs_c12\\\\test\\\\fixture\\\\config.ts"'
}
```